e2e/bitrise.yml

```diff
@@ -40,7 +40,7 @@ workflows:
     - path::./:
         title: Execute step
         inputs:
-          - ndk_revision: 21
+          - ndk_version: 21.1.6352462
```

Comment:
Couldn't we make this backward compatible if we use `sdkmanager --list` and pick the NDK which is the latest that share the major ? just thinking

Reply:
I'm not sure it's a good idea, those who need NDK either:
- are fine with the preinstalled version
- want a specific version, not just a matching major version